.github/workflows/release.yml

```diff
@@ -194,8 +194,18 @@ jobs:
           cat <<EOF > pr-body.txt
           ### Next steps
           - Test the release
-          - If this is an 'alpha' release, merge the PR **only** if the release job succeeds and the link checker is happy. Otherwise, close the PR without merging and cherrypick to main as required.
-          - Otherwise:
+          - For alpha releases:
+            - [ ] Should the GH release be published for this alpha? Make a call (give extra love to alphas deployed to Rerun Cloud or when external testing is required).
+              - If yes:
+                - [ ] Edit and publish the GitHub release
+                - [ ] Stretch goal: generate a raw changelog and add it to the GH release with this disclamer:
+                      **DISCLAIMER**: This is an unreviewed, automatically generated changelog. We only provide fully reviewed changelogs for final releases.
+              - If no:
+                - [ ] Delete the GH release draft
+            - [ ] Merge or close the release PR
+              - **IFF** the release job succeeds and the link checker is happy, you may merge the PR.
+              - Otherwise, close the PR without merging and cherrypick to `main` as required.
```

Comment:
should probably clarify what should be cherry-picked. Is the version patching done by the bot something that has to be cherry-picked?